libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset.component.spec.ts

```diff
@@ -428,6 +428,34 @@ describe('Vertical tabset component', () => {
     flush();
   }));
 
+  it('should focus the active tab when the tab container is opened on mobile view', fakeAsync(() => {
+    mediaQueryController.setBreakpoint('xs');
+    const fixture = createTestComponent();
+    fixture.detectChanges();
+
+    const el = fixture.nativeElement;
+
+    // click show tabs button to open tab group
+    let showTabsButton = el.querySelector('.sky-vertical-tabset-show-tabs-btn');
+    showTabsButton.click();
+    fixture.detectChanges();
+
+    // click second tab in second group
+    clickGroupButton(fixture, 1);
+    clickTab(fixture, 1, 1);
+
+    // open tab group
+    showTabsButton = el.querySelector('.sky-vertical-tabset-show-tabs-btn');
+    showTabsButton.click();
+    fixture.detectChanges();
+
+    const tabsContainer = getTabsContainer(fixture);
+
+    SkyAppTestUtility.fireDomEvent(tabsContainer, 'focus');
+    expect(elementHasFocus(getTab(fixture, 1, 1))).toBeTrue();
+    flush();
```

Comment:
Is this `flush` needed? Doesn't Angular 19 do that automatically?

Reply:
this is going into the 11 branch so its still on 18, i can add a comment to remove this in the future?

Reply:
i removed the flush entirely with a tweak to the unit test